packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts

```diff
@@ -155,7 +155,7 @@ export const planTabInfoContent = {
   },
   premium: {
     typography:
-      'Premium CPU instances guarantee a minimum processor model, AMD Epyc\u2122 7713 or higher, to ensure consistent high performance for more demanding workloads.',
+      'Premium CPU instances guarantee a minimum processor generation of AMD EPYC\u2122 Milan or newer, to ensure consistent high performance for more demanding workloads.',
```

Comment:
```suggestion
      'Premium CPU instances guarantee a minimum processor generation of AMD EPYC\u2122 Milan or newer to ensure consistent high performance for more demanding workloads.',
```

One more tweak here, I think -- we shouldn't need a comma after "to" and before "ensure" either; it breaks the flow of the sentence.

Reply:
I agree